src/main/java/com/sarality/action/NavigationStyleEnum.java

```diff
@@ -0,0 +1,25 @@
+package com.sarality.action;
+
+import android.content.Intent;
+
+/**
+ * Defines the different types of navigation behaviours that are available to an application at the
+ * time of starting an activity
+ *
+ * @author satya@ (Satya Puniani)
+ */
+public enum NavigationStyleEnum {
```

Comment:
Can we just call it NavigationStyle. Should use Enum at the end only if there is a conflict with original name like Boolean being called BooleanEnum